exiftool.go

```diff
@@ -155,17 +187,21 @@ func (e *Exiftool) ExtractMetadata(files ...string) []FileMetadata {
 			continue
 		}
 
+		e.log("sending to exiftool STDIN:")
```

Comment:
I'd prefer to avoid "skipping" possible error when calling `e.log` (like you did for the `Close` function).
On my mind, even if it is "just" logging, the user should know that something went wrong.
I didn't spot all the `e.log` and `e.logf` calls.

Reply:
I agree, silent failures aren't great. I thought it was the least bad option in this case. Alternatives are:
- log the error (seems like the wrong call because the logging already reported an error)
- add an error return value to the parent function (a breaking change)
- panic (is this the right way to report an error that is affecting a non-critical functionality? for example if the call would otherwise return successfully)

I'm happy to go with the option you prefer (or another idea if you have one), but the silent failure seemed like the best call to me in this particular case.

Reply:
What about storing the logging error directly in the concerned FileMetadata ?

Reply:
I added directly to FileMetadata. What do you think?